packages/metascraper-citation/README.md

```
@@ -0,0 +1,36 @@
+  <img style="width: 500px; margin:3rem 0 1.5rem;" src="https://metascraper.js.org/static/logo-banner.png" alt="metascraper">
+  <br>
+  <br>
+  <p align="center"><strong>metascraper-citation</strong>: Get title, author, date, and publisher from Highwire Press / Google Scholar <code>citation_*</code> meta tags.</p>
+  <p align="center">See our <a href="https://metascraper.js.org" target='_blank' rel='noopener noreferrer'>website</a> for more information.</p>
+  <br>
+</div>
+
+## Install
+
+```bash
+$ npm install metascraper-citation --save
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)